src/types/pagopa.ts

```diff
@@ -0,0 +1,403 @@
+import { fromNullable } from 'fp-ts/lib/Option';
```

Comment:
this will be hard to maintain in sync with the IO App types, probably these should belong to a common package ie. https://github.com/pagopa/io-pagopa-commons

Reply:
These types are "patches" to the generated types, due to the fact that the specs are wrong - there is a parallel stream of work to fix the specs so we can have the correct types generated and we can get rid of this file (and the same in io-app)